src/Handler/CreateOnly.php

```diff
@@ -68,10 +68,12 @@ public function __construct(
      */
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
-        $request = $request->withAttribute(
-            IdentifierGenerator::class,
-            $this->identifierGenerator->generate()
-        );
+        if ($request->getAttribute(IdentifierGenerator::class) === null) {
```

Comment:
This should have the same behaviour as `CreateAndFetch` regarding identifier generation.

Reply:
Yes of course, this was just a WIP to get some early feedback.